website/www/site/content/en/contribute/_index.md

```diff
@@ -24,366 +24,165 @@ See the License for the specific language governing permissions and
+
+## The Traits of an Apache Beam Committer
+
+{{< contributor/row_of_traits committer_traits >}}
+
+## Become a Project Management Committee (PMC) Member
+
+The PMC is the official managing body of the project as per the Apache Software Foundation governance model: “The Apache Way”. PMC members vote to set Apache Beam’s community and technical direction and are directly responsible for overseeing releases and the healthy development of the community.
+
+PMC members are also responsible for ensuring their project follows certain core requirements set by the board or other corporate officers of the ASF, like following Legal, Branding, and Infrastructure related requirements, along with ensuring their community operates within the [ASF Code of Conduct](https://www.apache.org/foundation/policies/conduct#code-of-conduct).
+
+Since the PMC members are responsible for establishing the Apache Beam vision and leading our community to work in harmony to achieve it, we look for committed individuals who can perform the responsibilities of the PMC, as well as lead our community to fulfill its vision. This is why we strive to build a diverse PMC that complements skills and expertise sharing the same sets of values.
+
+Ideally, we find candidates among active community members that have shown initiative to shape the direction of Beam and have shown willingness to learn the official processes, such as how to create or verify for releases, how to ensure trademark compliance for events, among others. We also expect PMC candidates to be level-headed, constructive, supportive, and willing to “disagree and commit” at times.
+
+A PMC member is also a committer, so if they are not initiated already, they will automatically become committers when joining the PMC. Therefore, the traits of Apache Beam committers also apply to PMC members.
+
+## Hear from the Community
+
+“Knowledge sharing and collaboration is something that comes naturally in the community. Together, we can work with the most complex problems and just solve them.
+”Matt Casters, Co-founder @ Apache Hop
+
+“I really enjoy working with Beam. The understanding of its internals empowers you to fine-tune the open source, customize it. Beam is very powerful and the abstraction is its most significant feature.”
+Talat Ulyarer, Sr Principal Software Engineer @ Palo Alto Networks
+
+“My takeaway: everyone should join the [Apache Beam] mailing lists! This is where all the discussions happen, and it is a great place for learning. I also encourage everyone to attend and participate in conferences to meet and connect with the community.”
+Tobias Kaymak, Senior Data Engineer @ Ricardo
```

Comment:
`Tobias Kaymak, Senior Data Engineer @ Ricardo` should be on the right side
Also need to add photo

Reply:
done